packages/manager/apps/dedicated/client/app/dedicated/dedicated-server/clusters/cluster/cluster.constant.js

```diff
@@ -1,4 +1,6 @@
 export const STATUS = { DONE: 'DONE' };
+export const GUIDES_LINK =
```

Comment:
```suggestion
export const 3AZ_GUIDE_LINK =
```

Reply:
3AZ_GUIDE_LINK is not valide name for variable bc we can't start naming with number so I change to _3AZ_GUIDE_LINK